llpc/test/shaderdb/ObjStorageBlock_TestMemoryQualifier_lit.frag

```diff
@@ -22,7 +22,7 @@ void main()
 
 ; SHADERTEST-LABEL: {{^// LLPC}} SPIR-V lowering results
 ; SHADERTEST: %{{[0-9]*}} = call i8 addrspace(7)* (...) @lgc.create.load.buffer.desc.{{[0-9a-z]*}}(i32 1, i32 0, i32 0,
-; SHADERTEST: %{{[0-9]*}} = load atomic <4 x float>, <4 x float> addrspace(7)* %{{[0-9]*}} unordered, align 16
+; SHADERTEST: %{{[0-9]*}} = load atomic float, float addrspace(7)* %{{[0-9a-z.]*}} unordered, align 4
```

Comment:
`.` matches every character. To allow for names with `.` in them, I think you'd want to use `\.` instead.

Reply:
Only digits will be matched here. I change it to only match digits. Thanks.